PROTOTYPE_NOTES.md

```diff
@@ -0,0 +1,130 @@
+# Subgraph Batching Prototype
+Version: Draft 0.1
+
+These notes are designed to maximise the utility of evaluating the subgraph batching prototype both for the customers involved and for Apollo.
+
+Please read these notes carefully before:
+ - Using the prototype router 
+ - Reporting results back to Apollo
+
+The [subgraph batching design document](https://docs.google.com/document/d/1KlGgNCm1sQWc-tYs2oqHauMBIYdrtlpIELySZDAnISE/edit?usp=sharing) may be helpful in evaluating the prototype.
+
+## Using the prototype
+
+### Deployment
+
+#### Accessing the prototype
+
+You can choose to build from source or from one of our pre-built options.
+
+##### Source
+
+The prototype router is publicly available on the main router repo on branch `subgraph-batching-prototype`. It can be accessed and built as normally.
+
+##### Binary
+
+If you want to use a pre-built router, you can access the prototype assets as follows:
+| Artifact | Location |
+| -------- | -------- |
+| Architecture specific binaries | [REPLACE ME] |
+| Docker image | [REPLACE ME] |
+| Helm chart | [REPLACE ME] |
+
+Note: These assets are only available until [REPLACE ME]
+
+### Configuration
+
+Because this is a prototype, we can't update the main `router` documentation and we don't have a way to deliver documentation for prototypes. For this prototype these configuration notes should suffice.
+
+The prototype is based off the latest version of the router so non subgraph batching specific documentation is [as usual](https://www.apollographql.com/docs/router/)
+
+Subgraph batching may be enabled for either all subgraphs or named subgraphs as follows.
+
+#### All Subgraphs
+
+This snippet enables client-side and subgraph batching for all subgraphs.
+
+```yaml
+batching:
+  enabled: true
+  mode: batch_http_link
+  subgraph:
+    all:
+      enabled: true
+```
+Note: `batching` must be enabled and `all` must also be enabled. `batching` enables client-side batching and everything under the optional `subgraph` key configures subgraph batching.
+
+#### Names Subgraphs
+
+This snippet enables client-side and subgraph batching for only the name `accounts` subgraph.
+
+```yaml
+batching:
+  enabled: true
+  mode: batch_http_link
+  subgraph:
+    subgraphs:
+      accounts:
+        enabled: true
+```
+Note: `batching` must be enabled and `subgraphs` is a list of named subgraphs which must be enabled. `batching` enables client-side batching and everything under the optional `subgraph` key configures subgraph batching. If a subgraph is not named (i.e.: not present in the sbugraphs list) then batching is not enabled for that subgraph. A subgraph may also be present but if `enabled` is `false`, that will also disable batching for that subgraph.
```

Comment:
```suggestion
Note: `batching` must be enabled and `subgraphs` is a list of named subgraphs which must be enabled. `batching` enables client-side batching and everything under the optional `subgraph` key configures subgraph batching. If a subgraph is not named (i.e.: not present in the subgraphs list) then batching is not enabled for that subgraph. A subgraph may also be present but if `enabled` is `false`, that will also disable batching for that subgraph.
```

Reply:
What's the easiest way of enabling batching for all subgraphs. Can we say 

```yaml
batching:
  enabled: true
  mode: batch_http_link
  subgraph:
    all: true
```